db/migrate/20211103112548_set_new_proceeding_fields_to_not_null.rb

```diff
@@ -0,0 +1,7 @@
+class SetNewProceedingFieldsToNotNull < ActiveRecord::Migration[6.1]
+  def change
+    change_column_null(:proceedings, :matter_type, false)
```

Comment:
I assume this has to be separate to allow the data migration to run between?

Reply:
That is correct.